services/venice-server/src/main/java/com/linkedin/venice/grpc/VeniceGrpcServerConfig.java

```diff
@@ -0,0 +1,101 @@
+package com.linkedin.venice.grpc;
+
+import io.grpc.BindableService;
+import io.grpc.ServerCredentials;
+import io.grpc.ServerInterceptor;
+import java.util.Collections;
+import java.util.List;
+
+
+public class VeniceGrpcServerConfig {
+  private final int port;
+  private final ServerCredentials credentials;
+  private final BindableService service;
+  private final List<? extends ServerInterceptor> interceptors;
+
+  VeniceGrpcServerConfig(Builder builder) {
+    port = builder.port;
+    credentials = builder.credentials;
+    service = builder.service;
+    interceptors = builder.interceptors;
+  }
+
+  public int getPort() {
+    return port;
+  }
+
+  public ServerCredentials getCredentials() {
+    return credentials;
+  }
+
+  public BindableService getService() {
+    return service;
+  }
+
+  public List<? extends ServerInterceptor> getInterceptors() {
+    return interceptors;
+  }
+
+  @Override
+  public String toString() {
+    return "";
```

Comment:
would be good to add the service name and port instead of having an empty string